cfy_manager/main.py

```diff
@@ -550,24 +505,24 @@ def _print_finish_message():
 def print_credentials_to_screen():
     password = config[MANAGER][SECURITY][ADMIN_PASSWORD]
     manager_status_reporter_token = config.get(
-        MANAGER_STATUS_REPORTER, {}).get(TOKEN, None)
+        MANAGER_STATUS_REPORTER, {}).get(STATUS_REPORTER_TOKEN, None)
```

Comment:
You can remove the None-s, that's the default

Reply:
You're right, removed.